manifest.json

```diff
@@ -437,26 +423,12 @@
         "targetDirPrefix": "ckeditor-sharedspace-"
       },
       "ckeditorBootstrapCK": {
-        "version": "1.0",
+        "version": "1.0.0",
         "downloadFormat": "zip",
         "url": "http://download.ckeditor.com/bootstrapck/releases/bootstrapck_1.0_0.zip",
```

Comment:
Btw why are we using an http url and not https?

Reply:
I just copy-pasted the link from there website. Like I downloaded CKEditor zip file from here https://ckeditor.com/cke4/release/CKEditor-4.9.2. Its http by default. But I checked now https works too.

Reply:
OK. Generally go with https. Follow our manifest file, not their website.

Reply:
Done.